Readme.md

```diff
@@ -3,3 +3,48 @@
 * [Default.aspx.cs](/CS/ASPxPivotGridLayoutUpgradeEvent/Default.aspx.cs) / [Default.aspx.vb](/VB/ASPxPivotGridLayoutUpgradeEvent/Default.aspx.vb)
 
 # ASPxPivotGrid - How to manage Pivot Grid Layout
+
+The Pivot Grid layout determines the position and appearance of its [visual elements](https://docs.devexpress.com/AspNet/3585/aspnet-webforms-controls/pivot-grid/ui-elements). You can customize and save the current Pivot Grid control's layout and apply the same settings to other Pivot Grid controls.
+
+The following example demonstrates how to save the layout from the one Pivot Grid and apply it to another Pivot Grid.
+
+The old Pivot Grid contains the "Year" and "Quarter" fields that do not exist in the new Pivot Grid. The new Pivot Grid contains the "Country" and "Sales Person" fields that do not exist in the old Pivot Grid. When you apply the saved layout to the new Pivot Grid control, the "Year" and "Quarter" fields from the saved layout replace the "Country" and "Sales Person" fields. This behavior is set by the [RemoveOldColumns](https://docs.devexpress.com/CoreLibraries/DevExpress.Utils.OptionsColumnLayout.RemoveOldColumns) and [AddNewColumns](https://docs.devexpress.com/CoreLibraries/DevExpress.Utils.OptionsColumnLayout.AddNewColumns) properties. In this example, the **RemoveOldColumns** and **AddNewColumns** properties are set to **false**. As a result, the Pivot Grid restores the fields that exist in the saved layout, and leaves fields that exist both in the Pivot Grid and in the saved layout with the same **Name** values. 
+
+The [LayoutVersion](https://docs.devexpress.com/CoreLibraries/DevExpress.Utils.OptionsLayoutBase.LayoutVersion) property is defined both for two Pivot Grid controls. When you click the "Restore layout" button and the saved layout is restored from a data store, the **LayoutUpgrade** event is raised because the layout versions are different. As a result, a new "Quantity" field is added to the new Pivot Grid control.
```

Comment:
The [LayoutVersion](https://docs.devexpress.com/CoreLibraries/DevExpress.Utils.OptionsLayoutBase.LayoutVersion) property is defined for two Pivot Grid controls. When you click the "Restore layout" button (and the saved layout is restored from a data store), the **LayoutUpgrade** event is raised since the layout versions are different. As a result, a new "Quantity" field is added to the new Pivot Grid control.